portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,14 +47,21 @@ public void init() {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Can you make this method name more descriptive for what it's doing?

Reply:
I don't think that I can change the method name because it's overriding a method from the HttpServlet class (the parent class to my DataServlet.java class). I just added a comment above the function to explain what it does. Does that help?

Reply:
Thanks. See my comment below. Although, keeping the bit explaining the random message won't hurt. 
nit: we typically add a space after "//" in comments. It improves the readability.

Reply:
Done!